src/Raven.Server/Documents/Indexes/Static/IndexAndTransformerCompiler.cs

```diff
@@ -28,7 +29,7 @@ namespace Raven.Server.Documents.Indexes.Static
     [SuppressMessage("ReSharper", "ConditionIsAlwaysTrueOrFalse")]
     public static class IndexAndTransformerCompiler
     {
-        private static readonly bool EnableDebugging = false; // for debugging purposes
+        private static readonly bool EnableDebugging = true; // for debugging purposes
```

Comment:
https://github.com/ayende/ravendb/blob/v4.0/test/SlowTests/Smuggler/LegacySmugglerTests.cs#L37

This test created 589 indexes and 71 transformers.

EnableDebugging (debug profile):
False => 15-16s
True => 17-18s